### DISCUSSION TITLE
what did I wrong? Engine-Model

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#4559] [Q&A] [answered]
hello, today the new engine model etc was announced, but after updating the experimental, and also switch to Developer Version, the EGT problem is back there (413°C with ENG off), also confusing Oil Press values like before. I think the EGT bug was solved in the last versions.

**Comment by ExampleWasTaken** [comment, marked as answer]
As mentioned [here](https://github.com/flybywiresim/a32nx/tree/master/docs#engines), the engine model is not yet complete and therefore has unrealistic values in the mentioned situations/conditions.
